src/client/datascience/jupyter/liveshare/hostJupyterServer.ts

```diff
@@ -175,24 +175,6 @@ export class HostJupyterServer implements INotebookServer {
         this.sessionManager = (await this.sessionManagerFactory.create(
             launchInfo.connectionInfo
         )) as JupyterSessionManager;
-
-        const defaultKernel: DefaultKernelConnectionMetadata = {
-            kind: 'startUsingDefaultKernel',
-            id: ''
-        };
-        // Try creating a session just to ensure we're connected. Callers of this function check to make sure jupyter
-        // is running and connectable.
```

Comment:
```Callers of this function check to make sure jupyter is running and connectable```

I assume this isn't true anymore (or is verified by other means)? I think this comment explains why we did this originally.

Reply:
We'll fail when we try to start a kernel or the like.
At that point we'll have some information.
Today nothing works, basically things fall over silently, this way, even if jupyter server isn't working, we can at least try to run a kernel & we'll get errors at that point in time.